unittests/test_using_virtual_board/test_from_list_connector.py

```diff
@@ -78,16 +78,6 @@ def test_simple(self):
         ]
         self.check_other_connect(as_list)
 
-    def test_list_too_big(self):
-        as_list = [
```

Comment:
Either it work and the test stays or it does not and we check for a good error

Reply:
Restored as above!